Makefile.am

```diff
@@ -73,12 +73,19 @@ $(ALL_KEYRINGS):
 	keys='$^'; \
 	for key in $$keys; do \
 	  $(GPG) --homedir gnupg --no-default-keyring \
-	    --keyring $(builddir)/$@ --import $$key || exit 1; \
+	    --keyring $(builddir)/$@.tmp --import $$key || exit 1; \
 	done
+	$(GPG) --homedir gnupg --no-default-keyring \
+	  --keyring $(builddir)/$@.tmp --export > $(builddir)/$@
```

Comment:
@dbnicholson hrmm...  good point - I guess I didn't notice the backtick on the `for`